BackEnd/src/main/java/com/unosquare/admin_core/back_end/service/EventService.java

```diff
@@ -99,6 +99,15 @@ public void cancelEvent(int eventId){
         }
     }
 
+    public void rejectEvent(int eventId){
```

Comment:
surely you're building an EventDto from the RejectHolidayViewModel, would you not pass that in and map it down to the Event?